CHANGELOG.md

```diff
@@ -27,3 +27,5 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Documentation
 ### Maintenance
 ### Refactoring
+* Add jni interface to use a binary hnsw index with faiss [#1747](https://github.com/opensearch-project/k-NN/pull/1747)
```

Comment:
Should this be added in Enhancements or Features?

Reply:
Hmm. There is no enhancement or feature where end user can benefit from this PR as of now. Therefore I put it under refactoring.